packages/mobile/src/send/SendAmount.tsx

```diff
@@ -87,7 +87,12 @@ export const sendAmountScreenNavOptions = ({
   return {
     ...emptyHeader,
     headerLeft: () => <BackButton eventName={eventName} />,
-    headerTitle: () => <HeaderTitleWithBalance title={title} token={CURRENCY_ENUM.DOLLAR} />,
+    headerTitle: () =>
+      route.params?.isOutgoingPaymentRequest ? (
+        <HeaderTitleWithSubtitle title={title} subTitle={' '} />
```

Comment:
In this case we're using a `HeaderTitleWithSubtitle` with an empty subtitle. I think it makes more sense to just pass the title directly if it's a payment request. It doesn't even need to be a function, you can do something like:
```
headerTitle: isOutgoingPaymentRequest ? title : () => (<HeaderTitleWithBalance ...>)
```

Reply:
+1, with the empty subtitle, the vertical alignment of the title is not right.

Reply:
thanks! changed